lib/watchable-message-store/WatchableMessageStore.d.ts

```diff
@@ -46,6 +46,7 @@ export declare class WatchableMessageStore {
     private bcc?;
     private dbName;
     private objStoreName;
+    lastCursorId: number;
```

Comment:
Nit: I’d prefer this `private` and a `resetCursor()` method.

Reply:
Done